pkg/sink/observer/observer.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package observer
+
+import (
+	"context"
+	"strings"
+
+	"github.com/pingcap/tiflow/cdc/contextutil"
+	"github.com/pingcap/tiflow/pkg/config"
+	"github.com/pingcap/tiflow/pkg/sink"
+	pmysql "github.com/pingcap/tiflow/pkg/sink/mysql"
+)
+
+// Observer defines an interface of downstream performance observer.
+type Observer interface {
+	Tick(ctx context.Context) error
```

Comment:
Perhaps `ObserveOnce` is a more semantically explicit method name?

Reply:
Still keep the `Tick` API, and add comments to this API about what it does.